website_sale_cache/doc/index.rst

```diff
@@ -15,13 +15,10 @@ The module does not need to be pre-configured.
 
 Usage
 =====
-When category display is activated, module immediately begins to cache the product categories,
-significantly speeding up the loading of the page.
+Enable the display of product categories on the site, through the customize menu in the upper right corner.
+Once the categories are cached after first load, the page load speed will be increased.
```

Comment:
Enable the display of product categories on website by using the customize menu at the upper right corner.
Once the categories are cached after first load, the page loading speed will be increased.